File: Main (2).java
/******************************************************************************

                            Online Java Compiler.
                Code, Compile, Run and Debug java program online.
Write your code in this editor and press "Run" button to execute it.

*******************************************************************************/
import java.util.*;
public class Main
{
	public  static void main(String[] args) {
		Scanner ob=new Scanner(System.in);
		int n=ob.nextInt();
		int i,f=0,l=n-1;
		int A[]=new int[n];
		for (i=0;i<n;i++)
		{
		    A[i]=ob.nextInt();
		}
	      Quicksort(A,f,l);
		System.out.println("Sorted Array is");
			for (i=0;i<n;i++)
		{
		   System.out.print(" "+A[i]);
		}
		
	}
    static	void Quicksort(int A[],int f,int l)
	{
	    if (f<l)
	    {
	        int q=Partition(A,f,l);
	        Quicksort(A,f,q-1);
	        Quicksort(A,q+1,l);
	    }
	}
	static int Partition(int A[],int f,int l)
	{
	    int x=A[l];
	    int i=f-1;
	    for(int j=f;j<l;j++)
	    {
	        if(A[j]<=x)
	        {   
	            i=i+1;
	            int temp=A[j];
	            A[j]=A[i];
	            A[i]=temp;
	        }
	    }
	    int temp=A[l];
	    A[l]=A[i+1];
	    A[i+1]=temp;
	   
	    return i+1;
	}
	
}

File: main (2).c
/******************************************************************************

                            Online C Compiler.
                Code, Compile, Run and Debug C program online.
Write your code in this editor and press "Run" button to compile and execute it.

*******************************************************************************/
#include <conio.h>
#include <stdio.h>

int main()
{
    printf("Enter size of array");
    int n,t;
    scanf("%d",&n);
    int arr[n+1];
    printf("Enter array");
    for(int i=0;i<n;i++)
    {
        scanf("%d",&arr[i]);
    }
     printf("Enter position in which no. is to be inserted");
     scanf("%d",&t);
     for(int i=0;i<n;i++)
    {
       if(i==t-1)
       {
        for(int i=n+1;i>=t-1;i--)
        {
            arr[i+1]=arr[i];
        }
         printf("Enter no. is to be inserted");
         scanf("%d",&arr[t-1]);
       }
    }
    for(int i=0;i<=n;i++)
    {
       printf("%d",arr[i]);  
    }
    getch();
    return 0;
}